modules/tensor_mechanics/doc/content/source/utils/RankTwoScalarTools.md

```diff
@@ -0,0 +1,247 @@
+# Rank Two Scalar Tools
+
+
+## Description
+
+Calls to this class calculates the following strain/stress quantities.
```

Comment:
I would say this instead:

This is a utility class that provides a set of functions to compute scalar quantities for stress or strain tensors. This  class is not directly invoked in the input file, but is called by several other classes that are, including <provide list of links to them>.